src/plugins/initiatives/plugin.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+import type {Plugin, PluginDirectoryContext} from "../../api/plugin";
+import type {PluginDeclaration} from "../../analysis/pluginDeclaration";
+import {parser, type InitiativesConfig} from "./config";
+import {declaration} from "./declaration";
+import {join as pathJoin} from "path";
+import type {ReferenceDetector} from "../../core/references";
+import type {WeightedGraph} from "../../core/weightedGraph";
+import {weightsForDeclaration} from "../../analysis/pluginDeclaration";
+import {createWeightedGraph} from "./createGraph";
+import {
+  type PluginId,
+  fromString as pluginIdFromString,
+} from "../../api/pluginId";
+import {loadJson} from "../../util/disk";
+import {loadDirectory as _loadDirectory} from "./initiativesDirectory";
+import * as Weights from "../../core/weights";
+
+async function loadConfig(
+  dirContext: PluginDirectoryContext
+): Promise<InitiativesConfig> {
+  const dirname = dirContext.configDirectory();
+  const path = pathJoin(dirname, "config.json");
+  return loadJson(path, parser);
+}
+
+const DIRECTORY_ENV_VAR_NAME = "SOURCECRED_INITIATIVES_DIRECTORY";
+
+function getDirectoryFromEnv(): string {
+  const rawDir = process.env[DIRECTORY_ENV_VAR_NAME] || "initiatives";
+  if (rawDir == null) {
+    throw new Error(
+      `No initiatives directory provided: set ${DIRECTORY_ENV_VAR_NAME}`
+    );
+  }
+  return rawDir;
+}
```

Comment:
Let's delete this function -- no reason to be getting stuff from ENV when we now have plugin directory contexts